test/builder_test.py

```diff
@@ -58,6 +59,11 @@ class TestDispenseBuilders(object):
     def test_column(self):
         for column in self.columns_reference:
             assert DispenseColumn(**column) == Dispense.builders.column(**column)
+            assert (
+                asdict(DispenseColumn(**column))
+                == asdict(Dispense.builders.column(**column))
+                == {"column": column["column"], "volume": str(column["volume"])}
+            )
```

Comment:
nice, we're asserting that DispenseColumn serializes properly here, but should we also include something like this in a test for the Dispense Instruction? 

Currently, [my PR](https://github.com/autoprotocol/autoprotocol-python/pull/388) makes the change that calls asdict on DispenseColumn inside the dispense method.

I guess I could add my commit to this branch? Just LMK!